src/components/cards/BalanceCard.tsx

```diff
@@ -27,12 +27,14 @@ import {
 } from '../modals/WalletListModal'
 import { Airship } from '../services/AirshipInstance'
 import { cacheStyles, type Theme, useTheme } from '../services/ThemeContext'
+import { UnscaledText } from '../text/UnscaledText'
 import { EdgeText } from '../themed/EdgeText'
 import { EdgeCard } from './EdgeCard'
 
 // Numbers larger than this are likely to overflow the display width so don't
 // use regular Text components which can auto shrink
 const MAX_ANIMATED_AMOUNT = '10000000'
+const ZERO_STRING = '0'
```

Comment:
This is because of the ESLint warning, right? Otherwise I would say that this constant is pretty inane and useless.